Atcoder_#358/b_ticket_counter.rb

```diff
@@ -0,0 +1,15 @@
+n, a = gets.chomp.split.map(&:to_i) 
+
+t = gets.chomp.split.map(&:to_i) 
+
+count_time = 0 
+
+# 各人がチケットを購入し終わる時間を計算
+n.times do |i|
+  if count_time <= t[i]
```

Comment:
```suggestion
  # 並んでいる人がいないとき
  if count_time <= t[i]
```
こんな感じでコメントを残すと、
このソースコードを後から読んだ人や、
1年とか経過した後に自分で読み返すときに役に立ちます。

Reply:
ありがとうございます！
こちらコメントを追加いたしました！
79813eb521292becef649ea61a6ad02f99f5aee7
また、今後は必ずコメントを入れるようにします！